CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ endif()
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 if(NOT MSVC)
-  set(CMAKE_C_STANDARD 99)
```

Comment:
@chuckatkins I wonder if it would be appropriate to move to C11, among other nice changes the thread library is a big win introduced there since it makes our concurrent code compatible across any other platform.

Reply:
Nevermind, I have checked that the C11 threading lib requires glibc 2.28 for GCC/Clang which in turn makes ubuntu bionic (18.04) unsupported. Lets come back to this when we drop support to ubuntu 18.04.

Any other C11 feature/lib is implemented since very old glibc/gcc/clang versions. It might just be worth to keep this change.